app/views/shared/_alerts.html.erb

```diff
@@ -0,0 +1,9 @@
+  <% if flash[:notice] %>
+  <div class="alert-success alert-dismissible fade show" role="alert">
+    <strong><%= notice %></strong>
+  </div>
+  <% else flash[:alert] %>
+  <div class="alert-warning alert-dismissible fade show" role="alert">
+    <strong><%= alert %></strong>
+  </div>
+  <% end %>
```

Comment:
There should be absolutely NO logic in views. View-related logic can be encapsulated in View helpers (app/helpers/*_helper.rb)